projects/dynamicforms/src/core/services/obo-customer/obo-customer.service.ts

```diff
@@ -0,0 +1,46 @@
+import { Injectable, OnDestroy } from '@angular/core';
+import { EventService, LogoutEvent, UserIdService } from '@spartacus/core';
+import { FSUser } from 'fsastorefrontlib/occ';
+import { BehaviorSubject, combineLatest, Observable, Subscription } from 'rxjs';
+import { map } from 'rxjs/operators';
+
+@Injectable({ providedIn: 'root' })
+export class OboCustomerService implements OnDestroy {
+  private subscription: Subscription = new Subscription();
+  private selectedCustomer: BehaviorSubject<FSUser> = new BehaviorSubject(null);
```

Comment:
`selectedCustomer` -> `selectedCustomerSource` so it's super clear that it's a behavior subject on the first glance (and because we implemented it like that in most of the code 🙂)